sequel/benchmarks/driver.rb

```diff
@@ -0,0 +1,194 @@
+      "benchmark_run[result][without_prepared_statements]" => without_prepared_statements["iterations_per_second"].round(3),
+      'benchmark_result_type[name]' => 'Number of iterations per second',
+      'benchmark_result_type[unit]' => 'Iterations per second'
+    })
+
+    submit_request(form_data, {
+      "benchmark_run[result][with_prepared_statements]" => with_prepared_statements["total_allocated_objects_per_iteration"],
+      "benchmark_run[result][without_prepared_statements]" => without_prepared_statements["total_allocated_objects_per_iteration"],
+      'benchmark_result_type[name]' => 'Allocated objects',
+      'benchmark_result_type[unit]' => 'Objects'
+    })
+  end
+
+  def generate_request
+    request = Net::HTTP::Post.new('/benchmark_runs')
+    request.basic_auth(ENV["API_NAME"], ENV["API_PASSWORD"])
+    request
+  end
+
+  def default_form_data(output, path, database)
+    data = {
+      'benchmark_type[category]' => output["label"],
+      'benchmark_type[script_url]' => "#{RAW_URL}#{Pathname.new(path).basename}",
+      'benchmark_type[digest]' => generate_digest(path, database),
+      'benchmark_run[environment]' => "#{`ruby -v`}",
+      'repo' => 'sequel',
+      'organization' => 'sequel'
+    }
+    
+    if(ENV['RAILS_COMMIT_HASH'])
+      data['commit_hash'] = ENV['RAILS_COMMIT_HASH']
+    elsif(ENV['RAILS_VERSION'])
+      data['version'] = ENV['RAILS_VERSION']
+    end
+    data
+  end
+
+  def submit_request(form_data, results)
+    request = generate_request
+    request.set_form_data(form_data.merge(results))
+    endpoint.request(request)
+  end
+
+  def files
+    Dir["#{File.expand_path(File.dirname(__FILE__))}/*"].select! { |path| path =~ /bm_.+/ }
+  end
+
+  def run_single(path, connection: nil, database: nil)
+    script = "RAILS_ENV=production ruby #{path}"
+
+    if connection
+      measure_sequel(script, connection, path, database)
+    else
+      output = measure(script)
+      return unless output
+      form_data = default_form_data(output, path, database)
+
+      submit_request(form_data, {
+        "benchmark_run[result][iterations_per_second]" => output["iterations_per_second"].round(3),
+        'benchmark_result_type[name]' => 'Number of iterations per second',
+        'benchmark_result_type[unit]' => 'Iterations per second'
+      })
+
+      submit_request(form_data, {
+        "benchmark_run[result][total_allocated_objects_per_iteration]" => output["total_allocated_objects_per_iteration"],
+        'benchmark_result_type[name]' => 'Allocated objects',
+        'benchmark_result_type[unit]' => 'Objects'
+      })
+    end
+    puts "Posting results to Web UI...."
+  end
+
+  def endpoint
+    @endpoint ||= begin
+      http = Net::HTTP.new(ENV["API_URL"] || 'rubybench.org', 443)
+      http.use_ssl = true
+      http
+    end
+  end
+
+  def generate_digest(path, database)
```

Comment:
I think we are benchmarking sequel for a specific version of rails, so rails comes off as a dependency. I cannot think of any other dependency for sequel.